numpy/core/code_generators/ufunc_docstrings.py

```diff
@@ -2541,6 +2542,128 @@ def add_newdoc(place, name, doc):
 
     """)
 
+add_newdoc('numpy.core.umath', 'matmul',
+    """
+    Matrix product of two arrays.
+
+    Parameters
+    ----------
+    x1, x2 : array_like
+        Input arrays, scalars not allowed.
+    out : ndarray, optional
+        A location into which the result is stored. If provided, it must have
+        a shape that matches the signature `(n,k),(k,m)->(n,m)`. If not
+        provided or `None`, a freshly-allocated array is returned.
+    **kwargs
+        For other keyword-only arguments, see the
+        :ref:`ufunc docs <ufuncs.kwargs>`.
+
+        ..versionchanged:: 1.16
+          Now handles ufunc kwargs
```

Comment:
If you're putting this under the `**kwargs` heading, then I think `.. versionadded::` would be more appropriate.

Reply:
The previous version accepted `out`, so I think this is a change.

Reply:
Right, but `out` has it's own argument section - this is just describing the other kwargs that aren't `out`, of which I believe all are new?

Reply:
changing